_security/configuration/security-admin.md

```diff
@@ -18,6 +18,9 @@ The first job of the script is to initialize the `.opendistro_security` index. T
 
 The script can be found at `/plugins/opensearch-security/tools/securityadmin.sh`. This is a relative path showing where the `securityadmin.sh` script is located. The absolute path depends on the directory where you've installed OpenSearch. For example, if you use Docker to install OpenSearch, the path will resemble the following: `/usr/share/opensearch/plugins/opensearch-security/tools/securityadmin.sh`.
 
+`securityadmin.sh` requires that SSL/TLS HTTP is enabled on your OpenSearch cluster. In other words, make sure that the `plugins.security.ssl.http.enabled: true` is set in `opensearch.yml` before proceeding. If you are not using SSL/TLS on HTTP layer in your cluster but still need to use `securityadmin.sh`, you can enable SSL/TLS on HTTP layer on a single node, for instance `ingest` node and run the `securityadmin.sh` on this node.
```

Comment:
```suggestion
`securityadmin.sh` requires that SSL/TLS HTTP is enabled for your OpenSearch cluster. In other words, make sure that your `opensearch.yml` has `plugins.security.ssl.http.enabled: true` before proceeding. If you are not using SSL/TLS on HTTP layer in your cluster but still need to use `securityadmin.sh`, you can enable SSL/TLS on HTTP layer on a single node, for instance `ingest` node and run the `securityadmin.sh` on this node.
```

Reply:
I would suggest explaining how to do the single node setup if you are going to mention it.

Reply:
Thanks @scrawfor99 I've added more details:
This can be achieved by configuring [REST layer TLS]({{site.url}}{{site.baseurl}}/security/configuration/tls/#rest-layer-tls) settings on only one node. OpenSearch restart on this node is necessary following the change in `opensearch.yml` file.